apollo/plugin.py

```diff
@@ -0,0 +1,175 @@
+        investors=event["investors"].split(", ") if event.get("investors") else [],
+        amount=event["amount"],
+        currency=event["currency"],
+    )
+
+
+def _parse_organization_data(org_data: dict) -> ApolloOrganization:
+    return ApolloOrganization(
+        name=org_data["name"],
+        website_url=org_data.get("website_url"),
+        blog_url=org_data.get("blog_url"),
+        angellist_url=org_data.get("angellist_url"),
+        twitter_url=org_data.get("twitter_url"),
+        facebook_url=org_data.get("facebook_url"),
+        phone=org_data.get("phone"),
+        founding_year=(
+            int(org_data["founded_year"]) if org_data.get("founded_year") else None
+        ),
+        industries=org_data.get("industries", []),
+        keywords=org_data.get("keywords", []),
+        estimated_num_employees=org_data.get("estimated_num_employees"),
+        street_address=org_data.get("street_address"),
+        city=org_data.get("city"),
+        state=org_data.get("state"),
+        country=org_data.get("country"),
+        seo_description=org_data.get("seo_description"),
+        annual_revenue=org_data.get("annual_revenue"),
+        funding_events=[
+            _parse_apollo_funding_event(event)
+            for event in org_data.get("funding_events", [])
+        ],
+    )
+
+
+def _parse_emploment(employment: dict) -> ApolloEmploymentHistory:
+    return ApolloEmploymentHistory(
+        organization_name=employment.get("organization_name"),
+        start_date=_parse_apollo_date(employment.get("start_date")),
+        end_date=_parse_apollo_date(employment.get("end_date")),
+        title=employment.get("title"),
+    )
+
+
+def _parse_people_enrichment_data(enrichment_data: dict) -> ApolloPersonProfile:
+    employment_history = [
+        _parse_emploment(employment)
+        for employment in enrichment_data.get("employment_history", [])
+    ]
+    organization = _parse_organization_data(enrichment_data.get("organization", {}))
+
+    return ApolloPersonProfile(
+        first_name=enrichment_data["first_name"],
+        last_name=enrichment_data["last_name"],
+        full_name=enrichment_data["name"],
+        linkedin_url=enrichment_data.get("linkedin_url"),
+        twitter_url=enrichment_data.get("twitter_url"),
+        github_url=enrichment_data.get("github_url"),
+        facebook_url=enrichment_data.get("facebook_url"),
+        headline=enrichment_data.get("headline"),
+        email=enrichment_data.get("email"),
+        state=enrichment_data.get("state"),
+        city=enrichment_data.get("city"),
+        country=enrichment_data.get("country"),
+        employment_history=employment_history,
+        organization=organization,
+        departments=enrichment_data.get("departments", []),
+        subdepartments=enrichment_data.get("subdepartments", []),
+        seniority=enrichment_data.get("seniority"),
+    )
+
+
+def apollo_people_enrichment(
+    first_name: str, last_name: str, email: str
```

Comment:
need to make these actually optional, and add more:

![image](https://github.com/d8e-ai/lutra-plugin/assets/1147434/d811ae81-836e-4fda-b1b4-181db6057757)

Reply:
@bozhi